podvm/qcow2/build-se-image.sh

```diff
@@ -0,0 +1,162 @@
+if [ "${SE_BOOT:-0}" != "1" ]; then
+    exit 0
+elif [ "${ARCH}" != "s390x" ]; then
+    echo "Build SE podvm image for $ARCH is not support yet"
```

Comment:
May be a better message is "Building of SE podvm image is only supported for s390x" ?

Reply:
done